content/docs/12-enterprise-version/03-deploying-palette-with-helm.md

```diff
@@ -320,6 +296,50 @@ If you notice that the pods in the `hubble-system` namespace are not initializin
 
 </WarningBox>
 
+
+# Upgrade Palette
+
+
+
+To upgrade Palette with a new Helm release, use the following steps. <br /> <br />
+
+1. Download the new version of the Helm Chart.
+
+
+
+2. Extract the new **values.yaml** file from the Helm Chart with the following command:
+
+    <br />
+
+    ```shell
+    tar xzvf /path/to/chart.tgz spectro-mgmt-plane/values.yaml
+    ```
+
+
+3. Compare the new **values.yaml** against the original **values.yaml** you used for the Palette installation. Address any new parameters added to the values file.
+
+
+
+
+4. Issue the following command to upgrade Palette. Use the same **values.yaml** file you used for the Palette installation. 
+
+    <br />
+
+    ```shell
+    helm upgrade palette /path/to/chart.tgz --file /path/to/orginal_values.yaml
+    ```
+ 
+
+## Post-Install Configuration Values
+
+The values you specified in the file **values.yaml**, more specifically, all the parameter values that fall under the parameter section `values.config` are stored in a configMap titled `configserver-cm`. 
+
+After the installation process, if you need to change any configuration values in the **values.yaml** file, you must use the Palette API to apply the change. You must use the API for Palette to consume the new system configuration values. Otherwise, internal system configurations stored in the Kubernetes configMap `configserver-cm` will not be updated. 
```

Comment:
```suggestion
After the installation process, if you need to change any configuration values under `values.config` in the **values.yaml** file, you must use the Palette API. Otherwise, internal system configurations stored in the Kubernetes ConfigMap `configserver-cm` will be updated, but Palette will not apply the new values. Once the initial installation has completed, Palette only applies changes to these configuration values if they submitted via API.
```